internal/exec/copy_glob.go

```
@@ -0,0 +1,471 @@
+// Named constants to avoid literal duplication.
+const (
+	logKeyPath        = "path"
+	logKeyError       = "error"
+	logKeyPattern     = "pattern"
+	shallowCopySuffix = "/*"
+
+	// finalTargetKey is used as a logging key for the final target.
+	finalTargetKey = "finalTarget"
+
+	// sourceKey is used as a logging key for the source.
+	sourceKey = "source"
```

golangci-lint (1.64.8)

[error] const `sourceKey` is unused

(unused)